R/test.R

```diff
@@ -180,3 +190,78 @@ print.shiny_runtests <- function(x, ..., reporter = "summary") {
 
   invisible(x)
 }
+
+
+#' Migrate legacy shinytest files to new test directory structure
```

Comment:
Question in general... should we support documentation of package names?

https://roxygen2.r-lib.org/articles/rd-formatting.html#character-formatting

```
\pkg{package_name}
```

Reply:
I personally don't really have strong feelings either way. It looks like, when rendered as HTML, it just results in bold text.